packages/ses/src/make-hardener.js

```diff
@@ -73,6 +75,10 @@ assert(getTypedArrayToStringTag);
  * @param {unknown} object
  */
 export const isTypedArray = object => {
+  if (!(object instanceof TypedArray)) {
+    // quicker reject for typical case
+    return false;
+  }
```

Comment:
This gives false negatives for inputs like `Object.setPrototypeOf(Uint8Array.from([]), null)`, which is acceptable for some applications of the exported function but possibly not for others. I don't think this change should be included.

Reply:
Well, damn. I agree with your diagnosis. But this early reject turned out to make a *huge* difference. How else could we get the benefit? I suppose we could put a try/catch around the freeze, and fall back to this only in the error case. Waddaya think?

Reply:
I did the try technique instead. PTAL. 

I am about to benchmark it so I can say how the three cases compare!

Reply:
Current master before this PR

![image](https://github.com/user-attachments/assets/a2fc4b6a-e414-4073-858f-a4552619cecf)
![image](https://github.com/user-attachments/assets/2beefc37-74e1-4569-b5ee-5d7835f9e97a)

With this PR, but before I took @gibson042 's suggestion of removing the `instanceof`
![image](https://github.com/user-attachments/assets/cfe3a8cd-77a5-47cf-bb66-a279cccf4b59)
![image](https://github.com/user-attachments/assets/98f83ebb-a59b-41a5-b156-54359d099f39)

With this PR after taking @gibson042 's suggestion, and using the try/catch trick instead
![image](https://github.com/user-attachments/assets/b56fbe9f-b1d8-45bc-bc0b-f43e67e239c2)
![image](https://github.com/user-attachments/assets/b5068a4f-ff8d-426f-ae43-b9aed0be1d44)

Reply:
To my great surprise, I conclude from this that both `harden` fixes were worse, which makes no sense to me. But that's why we measure. I will back out the harden fixes.

`passStyleOf` fix looks great! The new one takes about 76% of the original. I'll measure again after reverting the harden changes.

Reply:
After reverting the `harden` change

![image](https://github.com/user-attachments/assets/e4b43091-7376-48cf-9e66-30d07802296d) ![image](https://github.com/user-attachments/assets/5bbe2ed5-7d5b-4583-befc-4fd420546b1a)

Reply:
Scratch the 76%. The `passStyleOf` results are still significant enough to be worth it.

Altogether, I'm surprised at how much random variance I get. I would have thought 10,000 iterations was enough.

Reply:
Since I fully reversed the `harden` change, resolving.